test/buffer.test.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const t = require('tap')
+const t = require('node:test')
 const test = t.test
```

Comment:
I think we should match the other tests converted to `node:test` for consistency
```suggestion
const { test } = require('node:test')
```